common/src/main/java/bisq/common/persistence/db/FixedHash.java

```diff
@@ -0,0 +1,418 @@
+package bisq.common.persistence.db;
+
+import java.nio.ByteBuffer;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+
+import java.io.File;
+import java.io.RandomAccessFile;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author ebruno
```

Comment:
Where is this code copied from?

Reply:
Ah that whole `common/src/main/java/bisq/common/persistence/db/` dir should have been deleted. There is a later commit removing it... was from the MappedByteBuffer experiments.